Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -190,6 +191,21 @@ extension PhotonActionSheetProtocol {
             }
         }
 
+        let removeTopSitesPin = PhotonActionSheetItem(title: Strings.RemovePinTopsiteActionTitle, iconString: "action_unpin") { action in
+            guard let url = tab.url?.displayURL,
```

Comment:
I know this is just following the same pattern used above in the `pinToTopSites` closure. But I think we can clean both up now. 

Lets move this guard statement into one line